packages/react-devtools-shared/src/__tests__/inspectedElementContext-test.js

```diff
@@ -732,6 +737,8 @@ describe('InspectedElementContext', () => {
     );
     expect(object_of_objects.inner[meta.preview_short]).toBe('{…}');
 
+    expect(object_with_symbol['Symbol(name)']).toBe('hello');
```

Comment:
Would be worth adding some objects with inherited keys, so our tests at least codify the expected behavior.

Reply:
Ok, I'm going to add it.

Reply:
Done maybe, if I didn't forget any test.